feature-group-definitions/grid.yml

```diff
@@ -0,0 +1,30 @@
+spec: https://drafts.csswg.org/css-grid-3/
+caniuse: css-grid
+compat_features:
+  - css.properties.align-items
```

Comment:
Can we add the `minmax` function too? I don't know where it is in BCD right now though. Maybe it's already included in the `repeat` function which you've already added.
Also, the `auto-fit` and `auto-fill` values should be in there somewhere (they probably already are in the `repeat` function in fact).